cmd/ctr/commands/run/run_unix.go

```diff
@@ -29,12 +29,22 @@ import (
 	"github.com/containerd/containerd/contrib/seccomp"
 	"github.com/containerd/containerd/oci"
 	"github.com/containerd/containerd/platforms"
+	"github.com/containerd/containerd/runtime/v2/runc/options"
 	"github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/pkg/errors"
 	"github.com/urfave/cli"
 )
 
-var platformRunFlags []cli.Flag
+var platformRunFlags = []cli.Flag{
+	cli.StringFlag{
+		Name:  "runc-binary",
+		Usage: "specify runc-compatible binary",
```

Comment:
should this be OCI compatible?

Reply:
runc CLI spec is out of OCI

Reply:
So the shim name is `"io.containerd.runc.v2"`